readthedocs/projects/version_handling.py

```diff
@@ -109,7 +113,8 @@ def version_windows(versions, major=1, minor=1, point=1):
 def parse_version_failsafe(version_string):
     try:
         return Version(
-            unicodedata.normalize('NFKD', unicode(version_string)).encode('ascii', 'ignore')
+            six.text_type(
+                unicodedata.normalize('NFKD', six.text_type(version_string)))
```

Comment:
I'm not familiar with this conversion. I was under the assumption that `six.text_type` was a pointer to either `unicode()` or `str()`. If so, are we missing logic here for acsii encoding?